### DISCUSSION TITLE
[SOLVED] alternate path to database directory

### DISCUSSION THREAD

**Discussion by labatts** [martinrotter/rssguard#799] [Ideas]
Hi,

I would love for an option to point the directory to a different location.  This would allow me to sink things on pcloud / dropbox.  Thanks.

**Comment by akinokonomi** [comment]
See https://github.com/martinrotter/rssguard/issues/781

It's already possible to store all user data in a custom folder with `--data` command line option.

**Reply by labatts** [reply]
Thank you!  I appreciate the link too.  It helped me figure it out, even though it is pretty simple.  Thanks again.
